### PR TITLE
Cancelling already approved new asset pool

### PR DESCRIPTION
**Pull request by dimpar**
In case of some misconfiguration of a new asset pool, we give a governance an option to set already approved asset pool back to zero address.

### CODE REVIEW

File: test/AssetPool.test.js [hunk 1]
```diff
@@ -1190,6 +1191,60 @@ describe("AssetPool", () => {
...
+      await newUnderwriterToken.deployed()
+
+      const NewAssetPoolStub = await ethers.getContractFactory(
+        "NewAssetPoolStub"
+      )
+      newAssetPool = await NewAssetPoolStub.deploy(
+        collateralToken.address,
+        newUnderwriterToken.address
+      )
+      await newAssetPool.deployed()
```
Comment (tomaszslabon) [inline test/AssetPool.test.js:1214]: Could we move objects related to the new asset pool from `cancelNewPoolUpgrade` and `approveNewAssetPoolUpgrade` to the [main section of AssetPool.test.js](https://github.com/keep-network/coverage-pools/blob/951045379ce38f5e73fd61991f84a946a18b62fc/test/AssetPool.test.js#L25)? They are duplicated in both tests.

Comment (pdyraga) [reply test/AssetPool.test.js:1214]: I'd leave them as-is. It's true they are duplicated but given that they are duplicated only one time, in the context of `approveNewAssetPoolUpgrade` and the rest of the tests in this file do not need them, I think it's better to keep this code as-is.

**Review by pdyraga — APPROVED** [review]
LGTM, leaving the final decision to @tomaszslabon

File: test/AssetPool.test.js (hunk 1 above)
Comment (tomaszslabon) [reply test/AssetPool.test.js:1214]: @pdyraga I'm ok with leaving it as is.

**Pull request merged**